src/tests/ftest/util/server_utils.py

```diff
@@ -50,6 +58,487 @@ class ServerFailed(Exception):
+        self.report_uri = BasicParameter(None)             # Orterun param
+        self.enable_recovery = BasicParameter(None, True)  # Orterun param
+        self.export = BasicParameter(None)                 # Orterun param
+
+    @property
+    def hosts(self):
+        """Hosts attribute getter."""
+        return self._hosts
+
+    @hosts.setter
+    def hosts(self, value):
+        """Hosts attribute setter
+
+        Args:
+            value (tuple): (list of hosts, workdir, slots)
+        """
+        self._hosts, workdir, slots = value
+        self.runner.processes.value = len(self._hosts)
+        self.runner.hostfile.value = write_host_file(
+            self._hosts, workdir, slots)
+        self.runner.job.server_cnt = len(self._hosts)
+
+    def get_params(self, test):
+        """Get values from the yaml file and assign them respectively
+            to the server command and the orterun command.
+
+        Args:
+            test (Test): avocado Test object
+        """
+        server_params = ["debug", "sudo", "srv_timeout"]
+        server_start_params = ["attach", "insecure"]
+        runner_params = ["enable_recovery", "export", "report_uri"]
+        super(ServerManager, self).get_params(test)
+        self.runner.job.yaml_params.get_params(test)
+        self.runner.get_params(test)
+        for name in self.get_param_names():
+            if name in server_params:
+                if name == "sudo":
+                    setattr(self.runner.job, name, getattr(self, name).value)
+                elif name == "srv_timeout":
+                    setattr(self.runner.job, name, getattr(self, name).value)
+                else:
+                    getattr(
+                        self.runner.job, name).value = getattr(self, name).value
+            if name in server_start_params:
+                getattr(self.runner.job.action_command, name).value = \
+                    getattr(self, name).value
+            if name in runner_params:
+                getattr(self.runner, name).value = getattr(self, name).value
+
+    def run(self):
+        """Execute the runner subprocess."""
+        self.log.info("Start CMD>>> %s", str(self.runner))
+        return self.runner.run()
+
+    def start(self, yamlfile):
+        """Start the server through the runner."""
+        self.runner.job.set_config(yamlfile)
+        self.server_clean()
+
+        # Prepare nvme storage in servers
+        if self.runner.job.yaml_params.is_nvme():
+            self.log.info("Performing nvme storage prepare in <format> mode")
+            storage_prepare(self._hosts, "root")
+            self.runner.job.sudo = True
+
+            # Make sure log file has been created for ownership change
+            data_dict = self.runner.job.yaml_params.data
+            if data_dict and "log_file" in data_dict['servers'][0]:
+                self.log.info("Creating log file: {}".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments